lib/core/proxy.c

```diff
@@ -300,6 +314,27 @@ static h2o_httpclient_t *detach_client(struct rp_generator_t *self)
     return client;
 }
 
+static int empty_pipe(int fd)
+{
+    ssize_t ret;
+    char *dst = alloca(1024);
```

Comment:
This is equivalent to declaring a local variable of type `char [1024]`?

Reply:
Yes; but I assume from your comment that you prefer `char dst[1024]` instead, so I've removed it in commit 204471f47ebd6b8cf313208b840baafeb6ebd389 -- I'll mark this conversation as resolved.